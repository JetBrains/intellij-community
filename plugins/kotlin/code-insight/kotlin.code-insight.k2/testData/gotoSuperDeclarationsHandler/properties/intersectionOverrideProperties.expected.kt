Property: /I1.foo
Property: /I2.foo
