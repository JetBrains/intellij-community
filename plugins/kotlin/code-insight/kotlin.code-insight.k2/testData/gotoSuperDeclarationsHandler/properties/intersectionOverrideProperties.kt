interface I1 {
    val foo: Int
}

interface I2 {
    val foo: Int
}

interface I3 : I1, I2

class B : I3 {
    override val f<caret>oo: Int = 5

}