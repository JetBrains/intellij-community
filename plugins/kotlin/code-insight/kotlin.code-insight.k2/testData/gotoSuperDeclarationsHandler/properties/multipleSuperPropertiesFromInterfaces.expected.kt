Property: /A.foo
Property: /I.foo
