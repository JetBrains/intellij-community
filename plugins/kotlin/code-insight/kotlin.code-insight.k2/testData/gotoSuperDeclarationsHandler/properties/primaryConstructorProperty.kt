interface A {
    val pp: String
}

class B(override val p<caret>p: String) : A