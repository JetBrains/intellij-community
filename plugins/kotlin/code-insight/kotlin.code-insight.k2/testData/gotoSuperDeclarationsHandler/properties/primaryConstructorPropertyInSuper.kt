open class D(open val d: String = "") {}

class E : D() {
    override val <caret>d: String = ""
}