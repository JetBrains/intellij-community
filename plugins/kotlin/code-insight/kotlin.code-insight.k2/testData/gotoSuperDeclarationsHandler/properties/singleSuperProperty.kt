open class A {
    open val foo: Int
}

class B : A() {
    override val f<caret>oo = 10
}