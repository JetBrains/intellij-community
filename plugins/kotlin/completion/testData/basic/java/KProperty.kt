// FIR_COMPARISON
// FIR_IDENTICAL
/* Number of result in completion should be fixed after removing duplicates */
package first

fun firstFun() {
  val a = KProp<caret>
}

// INVOCATION_COUNT: 1
// EXIST: { lookupString:"KProperty", itemText:"KProperty", tailText:"<V> (kotlin.reflect)", icon: "org/jetbrains/kotlin/idea/icons/interfaceKotlin.svg"}
