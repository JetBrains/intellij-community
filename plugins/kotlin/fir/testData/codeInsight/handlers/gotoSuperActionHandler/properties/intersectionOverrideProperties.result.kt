KtProperty: I1.foo
KtProperty: I2.foo
