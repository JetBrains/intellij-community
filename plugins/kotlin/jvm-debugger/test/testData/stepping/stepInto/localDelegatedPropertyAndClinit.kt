import kotlin.properties.ReadOnlyProperty
import kotlin.reflect.KProperty

class MyDelegate : ReadOnlyProperty<Any?, String> {
    override fun getValue(thisRef: Any?, property: KProperty<*>) = "OK"
}

class MyClass {
    fun foo() {
        val x by MyDelegate()
    }
}

fun main() {
    // STEP_INTO: 2
    //Breakpoint!
    val y = MyClass()
}