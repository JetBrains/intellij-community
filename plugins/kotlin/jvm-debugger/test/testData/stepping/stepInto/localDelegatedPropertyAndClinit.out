LineBreakpoint created at localDelegatedPropertyAndClinit.kt:17
Run Java
Connected to the target VM
localDelegatedPropertyAndClinit.kt:17
localDelegatedPropertyAndClinit.kt:8
localDelegatedPropertyAndClinit.kt:17
Disconnected from the target VM

Process finished with exit code 0
